### PR TITLE
Sarthak

### PR DESCRIPTION
**Pull request by uaryal02**
pull request

### CODE REVIEW

**Review by uaryal02 — COMMENTED** [review]
private class variables should have an underscore suffix.

**Comment by Sarthakjo** [comment]
I am gonna fix the code. Thanks for reviewing.

File: ExitController.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import bcccp.carpark.ICarSensorResponder;
+import bcccp.carpark.ICarpark;
+import bcccp.carpark.IGate;
+import bcccp.tickets.adhoc.IAdhocTicket;
+
+public class ExitController 
+		implements ICarSensorResponder,
+		           IExitController {
+	
+	private IGate exitGate;
```
Comment (uaryal02) [inline ExitController.java]: try to separate class variables from method variables buy using _ suffix

File: ExitUI.java [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+import javax.swing.JButton;
+import java.awt.event.ActionListener;
+import java.awt.event.ActionEvent;
+import java.awt.Color;
+import javax.swing.UIManager;
+
+@SuppressWarnings("serial")
+public class ExitUI extends JFrame implements IExitUI {
+
+	private JPanel contentPane;
```
Comment (uaryal02) [inline ExitUI.java]: it is better to change the variable name from contentPane to contentPanel as java swing variable names need to end with their types.

**Review by uaryal02 — COMMENTED** [review]
Thank you for making the required changes

File: ExitController.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import bcccp.carpark.IGate;
+import bcccp.tickets.adhoc.IAdhocTicket;
+
+public class ExitController 
+		implements ICarSensorResponder,
+		           IExitController {
+	
+	private IGate exitGate_;
+	private ICarSensor insideSensor_;
+	private ICarSensor outsideSensor_;
```
Comment (uaryal02) [inline ExitController.java:16]: good to see that changes have been made as per the review.

**Review by uaryal02 — COMMENTED** [review]
Good to see changes as per the review

**Pull request closed without merging**